### ISSUE TITLE
needs a default configuration file

### ISSUE THREAD

**Issue by cdetrio** [wasmx/wasm-chisel#82]
The docs are wrong about the CLI tool.

> It comes with a configuration file chisel.yml.

**Comment by axic** [comment]
Should have a `chisel init` which creates a template file.

**Comment by cdetrio** [comment]
I'm still using chisel 0.1.0 because its the only version I can figure out how to use.

**Comment by axic** [comment]
Right, it is not documented well. Here's an example: https://github.com/ewasm/ewasm-precompiles/blob/master/chisel.yml
